package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "walletconnect-monorepo",
+  "version": "0.7.8",
```

Comment:
This needs to be removed, this package.json is not a package!!!

I've told you this repository is a wrapper to manage the other packages. There is no such thing a `walletconnect-monorepo` on NPM and there shouldnt be. Please look into monorepos further!